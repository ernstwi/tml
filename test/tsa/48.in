init a

[a, b] -> c -> d
    invar { x < 5 }
