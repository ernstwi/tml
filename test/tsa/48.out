Semantic error: Location property on edge [a, b] -> [c]
Semantic error: Location property on edge [c] -> [d]
